### PR TITLE
Add mustache to dev env

### PR DESCRIPTION
**Pull request by smaarn**
## Description

mustache is a very simple template language which could be used to easily generate internationalized wizards.

This is an attempt to integrate it in the dev environment and would be a prerequisite for https://github.com/SynoCommunity/spksrc/pull/5817

Homepage: https://mustache.github.io/

## Checklist

- [x] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [x] Includes small framework changes
- [x] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by smaarn** [comment]
TBH there is this still the documentation which could be added about the "how to use it".

**Comment by hgy59** [comment]
@smaarn I had to break the identation to limit the single RUN command to install packages.

and added zip package for a future removement of native/zip...

@th0ma7, @publicarray, @mreid-tt can you have a second look on the wizard generation from templates?

File: mk/spksrc.spk.mk
Comment (th0ma7) [inline mk/spksrc.spk.mk:1]: Nicely done.

**Review by hgy59 — APPROVED** [review]
LGTM

**Comment by hgy59** [comment]
@smaarn 
BTW normally I used to add a custom build step to install packages before they are available in the dev env with
```
   sudo apt-get update
   sudo apt-get install ruby-mustache
```

but due to the sudo file error (fixed here) this did not work in spksrc containers on my newer dev machine with debian bookworm.

**Comment by mreid-tt** [comment]
@smaarn, I see the wizards in cops and I like them. I'd like to incorporate in the next update to owncloud. Will the template also work for `install_uifile.sh` files?

**Pull request merged**